prime-router/docs/api/waters-delivery.yml

```diff
@@ -0,0 +1,86 @@
+openapi: "3.0.3"
+info:
+  title: ReportStream Delivery API
+  description: This API reflects the point of view of the STLTs and provides lookup functionality
+  version: "1.0"
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://open.cdc.gov
+    email: usds@cdc.gov
+  license:
+    name: Creative Commons Zero v1.0 Universal
+    url: https://github.com/CDCgov/prime-reportstream/blob/master/LICENSE
+servers:
+  - url: https://prime.cdc.gov/api
+    description: Production server (uses live data)
+  - url: http://localhost:7071/api
+    description: Developer Environment
+paths:
+  /api/waters/{organization}/deliveries:
+    get:
+      operationId: getAvailableReports
+      summary: Get available reports for a STLT
+      description: Retrieve a list of reports available for download by a STLT
+      security:
+        - OktaAuth: [ ANONYMOUS ]
+        - Server2Server: []
+      parameters:
+        - name: organization
+          in: path
+          description: receiver whose reports we want to see
+          required: true
+          schema:
+            type: string
+        - $ref: "./components/sort_and_filter.yml#/parameters/sortOrder"
+        - $ref: "./components/sort_and_filter.yml#/parameters/sortColumn"
+        - $ref: "./components/sort_and_filter.yml#/parameters/resultsAfterDate"
+        - $ref: "./components/sort_and_filter.yml#/parameters/resultsBeforeDate"
+        - $ref: "./components/sort_and_filter.yml#/parameters/pageSize"
+      responses:
+        '200':
+          description: a list of AvailableReport objects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AvailableReportsResponse'
+        '400':
+          description: bad request
+        '401':
+          description: unauthorized
+        '404':
+          description: organization does not exist
+components:
+  securitySchemes:
+    $ref: "./components/security_schemes.yml"
```

Comment:
https://editor.swagger.io/ has trouble with this line (and therefore the references on lines 25/26).

Should this be more like  `- $ref: "./components/sort_and_filter.yml#/parameters/sortOrder" ` in style?

Reply:
This happens because the parser doesn't understand separate files in some cases. If you use a bundler and then parse it the editor accepts it